Main.java

```diff
@@ -9,7 +11,53 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // Create a Scanner for user input 
+    Scanner input = new Scanner(System.in); 
+
+    // prompt the user to enter the amount of money they spent and create variable for the amount the user spent  
+    System.out.println("Please enter the amount the customer spent"); 
+    double amount_spent = input.nextDouble();  
+
+    // creating variables for amount spent with the disscount 
+    double First_Dis = amount_spent - (amount_spent/100)*10; 
+    double Second_Dis = amount_spent - (amount_spent/100)*20; 
+    double Third_Dis = amount_spent - (amount_spent/100)*30;
+    double Forth_Dis = amount_spent - (amount_spent/100)*40;
+
+    // creating variables to the amount saved with the disscount 
+    double A_save = (amount_spent/100)*10; 
+    double B_save = (amount_spent/100)*20;
+    double C_save = (amount_spent/100)*30;
+    double D_save = (amount_spent/100)*40; 
```

Comment:
Remember, variables should always be named using camelCase and describe what they are storing or being used for.  You have named these like a constant.